provision/testdata/list_flavors.stdout.txt

```diff
@@ -1,5 +1,6 @@
 INFO: Available configuration flavors:
 INFO: openshift-4.5-openstack:	Red Hat OpenShift Container Platform 4.5 on OpenStack
+INFO: openshift-4.5-esx:	Openshift 4.5 ESX
 INFO: openshift-4.4-openstack:	Red Hat OpenShift Container Platform 4.4 on OpenStack
 INFO: openshift-4.4-esx:	Openshift 4.4 ESX
 INFO: openshift-4.3:	Red Hat OpenShift Container Platform 4.3
```

Comment:
We don't support this flavor so we need to remove it. We also need to check in the release/jordan branch that we are not displaying this.